data_interrogator/interrogators.py

```diff
@@ -275,7 +282,7 @@ def get_annotation(self, column):
                 condition_key, condition_val = condition.split('=', 1)
                 conditions[normalise_field(condition_key)] = normalise_field(condition_val)
             annotation = self.available_aggregations[agg](field=field, **conditions)
-        elif agg == 'join':
+        elif agg == 'concat':
```

Comment:
Why the change?

Reply:
Because there is no "join" look up, this has been broken for a while.